index.html

```diff
@@ -40,7 +40,176 @@ <h4>I'am a Software Developer</h4>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+        <div class="pj1">
+          <h3 class="tproj">Profesional Art Printing Data</h3>
+          <p class="desproj">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="media2">
+            <li><div class="cont1">html</div></li>
+            <li><div class="cont1">bootstrap</div></li>
+            <li><div class="cont1">Ruby</div></li>
+          </ul>
+          <nav id="contbut">
+            <button class="bigbutton" class="button2" type="button">See Project</button>
+          </nav>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="about">
+      <section name="first_about">
+        <h1 id="t_about">About me</h1>
+        <p id="c_about">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div id="fit_cont">
+         <nav>
+            <button id="button_about" type="button">Get My Resume</button>
+          </nav>
+          <div class="bgimg2"></div>
+        </div>
+      </section>
+
+      <div class="divisor div_set"></div>
+
+      <section id="more_about">
+        <div class="center laguage">
+          <ul id="style1">
+            <li class="align">Language
+              <ul class="none">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+        
+        <div class="center frameworks">
+          <ul id="style2">
+            <li class="align">Frameworks
+              <ul class="none">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+
+        <div class="center skills">
+          <ul id="style3">
+            <li class="align">Skills
```

Comment:
- [ ] Good job so far 👏 Your design looks great 👏 However concerning this section, there is an improvement that I would suggest you make. I can see that you know how to use `nested list` however the way you separated technology categories in `div` is not good. It is suggested that they should be in one list `<ul>` where each is a list item `<li>`
EXAMPLE
```
<ul>
  <li>Framework
    <ul>
      <li>Ruby on Rails</li>
      <li>Bootstrap</li>
    </ul>
  </li>
  <li>Languages
    <ul>
      <li>Ruby</li>
      <li>Javascript</li>
    </ul>
  </li>
    ...
</ul>
```

Reply:
Hi @BertrandConxy.
I appreciate your help and suggestions!!!
I just want to know if it is optional or mandatory to make the changes on my unordered list. I will work on that, but if it is not necessary I could send my project without the navs on my buttons and submit this project.